app/views/provider_interface/start_page/show.html.erb

```diff
@@ -1,5 +1,8 @@
 <div class="govuk-width-container">
   <main id="main-content" class="govuk-main-wrapper govuk-main-wrapper--auto-spacing" role="main">
+    <%= render ProviderInterface::KeyDatesBanner.new %>
```

Comment:
Might be checking with designers the position of the banner on the start page before merging